### DISCUSSION TITLE
Do you guys will update your thing again?

### DISCUSSION THREAD

**Discussion by kazetenzo** [WakuWakuPadoru/WuWa_Simple_FPSUnlocker#14] [Q&A] [answered]
So i usually use ur thing to set my resolution and fps to the game but, after the recent updates from the game, i think there's bug that make the thing not working again, and i try to set the config manually from the .ini thing called "GameUserSetting.ini" but not make any diffrences, and stay at 1200p:"

**Comment by WakuWakuPadoru** [comment]
> So i usually use ur thing to set my resolution and fps to the game but, after the recent updates from the game, i think there's bug that make the thing not working again, and i try to set the config manually from the .ini thing called "GameUserSetting.ini" but not make any diffrences, and stay at 1200p:"

I will give it a look and investigate. Did you try reapplying using the latest version of the unlocker? That might be required at times if the game updates or modified the config file.

**Reply by kazetenzo** [reply]
i did reapplying using the lastest version for setting the resolution on 1280x800 but, theres no diffrence and stay at 1200p

**Reply by jakesway99** [reply]
It might be a change with how the game deals with the config files, doesn't work for me either even though the resolution in the settings is accurate.

**Reply by WakuWakuPadoru** [reply, marked as answer]
Thanks for the feedback. Will work on it :)

**Reply by WakuWakuPadoru** [reply]
> i did reapplying using the lastest version for setting the resolution on 1280x800 but, theres no diffrence and stay at 1200p

Very very late but the latest version works now with 1.20. Kuro revamped their DB structure a bit, so updated it to work! Tested too.

**Reply by jakesway99** [reply]
Thank you! glad it's working again the blurriness was really getting to me
